lib/std/math.zig

```diff
@@ -1117,13 +1117,12 @@ pub fn alignCast(comptime alignment: u29, ptr: anytype) AlignCastError!AlignCast
     return @alignCast(ptr);
 }
 
-/// Asserts `int > 0`.
 pub fn isPowerOfTwo(int: anytype) bool {
-    assert(int > 0);
-    return (int & (int - 1)) == 0;
+    return int > 0 and (int & (int - 1)) == 0;
```

Comment:
That’s not how this is supposed to be solved. This is similar to what my PR did originally.

https://github.com/ziglang/zig/issues/16021
https://github.com/ziglang/zig/pull/15888
https://github.com/ziglang/zig/pull/15888#pullrequestreview-1477785337

Reply:
I see. I didn't follow the chain of PRs long enough. Reverted.